pyomo/contrib/mindtpy/nlp_solve.py

```diff
@@ -10,21 +10,20 @@
 #  ___________________________________________________________________________
 
 """Solution of NLP subproblems."""
-from __future__ import division
-
 from pyomo.common.collections import ComponentMap
 from pyomo.common.errors import InfeasibleConstraintException
 from pyomo.contrib.mindtpy.cut_generation import (add_oa_cuts,
                                                   add_no_good_cuts, add_affine_cuts)
 from pyomo.contrib.mindtpy.util import add_feas_slacks, set_solver_options, update_primal_bound
 from pyomo.contrib.gdpopt.util import copy_var_list_values, get_main_elapsed_time, time_code, SuppressInfeasibleWarning
-from pyomo.core import (Constraint, Objective,
-                        TransformationFactory, minimize, value)
+from pyomo.core import Constraint, Objective, TransformationFactory, minimize, value
 from pyomo.opt import TerminationCondition as tc
 from pyomo.opt import SolverFactory, SolverResults, SolverStatus
 
 
 def solve_subproblem(solve_data, config):
+    # TODO: we will keep working_model.clone first
+    # This function is algorithm-dependent, therefore we will redefine it as method.
```

Comment:
Is this a typo? Should it be algorithm-independent?

Reply:
Yes. It's a typo.